src/status_im2/contexts/quo_preview/preview.cljs

```diff
@@ -330,28 +330,28 @@
      [common/navigation-bar]
      [rn/scroll-view
       {:style                           (style/panel-basic)
-       :shows-vertical-scroll-indicator false}
-      [rn/pressable {:on-press rn/dismiss-keyboard!}
-       (when descriptor
-         [rn/view {:style style/customizer-container}
-          [customizer state descriptor]])
-       (if blur?
-         [rn/view {:style (merge style/component-container component-container-style)}
-          (into [blur-view
-                 {:theme                 theme
-                  :show-blur-background? show-blur-background?
-                  :height                blur-height
-                  :style                 (merge {:width     "100%"
-                                                 :flex-grow 1}
-                                                (when-not show-blur-background?
-                                                  {:padding-horizontal 0
-                                                   :top                0})
-                                                blur-container-style)
-                  :blur-view-props       (merge {:blur-type theme}
-                                                blur-view-props)}]
-                children)]
-         (into [rn/view {:style (merge style/component-container component-container-style)}]
-               children))]]]))
+       :shows-vertical-scroll-indicator false
+       :keyboard-should-persist-taps    :handled}
```

Comment:
Removed pressable here because it was causing an issue: if a component with a scroll-view or flat-list would be rendered as children, they wouldn't get touch events

Reply:
hmm this looks fishy. This functionality is here for a reason. It's much safer to make this a configurable option. Are you sure this is safe for all other preview screens 🤔

Reply:
You mean to render the `rn/pressable` or not based on on prop, right?
I tested other preview screens that had inputs and seemed to work. On the other hand, before this change, no component would receive events. And being a preview screen it seemed like an acceptable risk, but maybe I'm wrong. I can add the configurable option

Reply:
Ok, I added this prop. I'm really unsure about this code, I'm not sure how you usually implement this pattern, so any comment is welcome